app/src/main/java/org/kiwix/kiwixmobile/main/MainActivity.java

```diff
@@ -511,6 +514,8 @@ private void hideTabSwitcher() {
     actionBar.setDisplayShowTitleEnabled(true);
 
     drawerLayout.setDrawerLockMode(DrawerLayout.LOCK_MODE_UNLOCKED);
+    closeAllTabsButton.setImageDrawable(
+        ContextCompat.getDrawable(getApplicationContext(), R.drawable.ic_close_white_24dp));
```

Comment:
I think you can just pass `this` as the `context`.

Reply:
@abdulwd Changes done. Please review.